test/example/test.dart

```diff
@@ -0,0 +1,83 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
```

Comment:
nit: for all new files update the year to 2019.

Reply:
will do